### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by poornima2025**
Hi @ramanji1990 ,

Created terraform for ec2 server creation. Can you please review and approve the code.

Regards,
Poornima

### CODE REVIEW

File: ec2.tf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+resource "aws_instance" "mytftesting" {
+  ami           = "ami-0763cf792771fe1bd"
+  instance_type = "t2.micro"
```
Comment (ramanji1990) [inline ec2.tf]: Hi @poornima2025 ,
can you please change instance_type from t2.micro to t2.large.

File: ec2.tf [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+  tags = {
+    Name = "mytftesting"
```
Comment (ramanji1990) [inline ec2.tf]: Can you change server name from mytftesting to test.

**Review by ramanji1990 — CHANGES_REQUESTED** [review]
Can you please work on review comments.

**Comment by ramanji1990** [comment]
Updated PR with new code change. Please review and approve my code.

**Review by poornima2025 — COMMENTED** [review]
Code looks good now.

**Pull request merged**